### PR TITLE
Revised the Download page UI

### PR DESCRIPTION
**Pull request by ievgend2**

### CODE REVIEW

**Review by Javier-Jimenez-18 — APPROVED** [review]
Looks good. Any screenshots to see the final views? I'm sure Ramiro will approve it when he sees the changes tomorrow (if he has time to meet).

**Review by kholtzman5610 — APPROVED** [review]
Styling looks better on all just add a few screenshots to show the deference otherwise everything is great.

File: app/views/pages/covid_19.html.erb [hunk 1]
```diff
@@ -14,16 +14,5 @@
 
     <p>For AACT database users, a table called "covid_19_studies" containing the 68 columns from the spreadsheet, is now included within AACT.</p>
```
Comment (micronix) [inline app/views/pages/covid_19.html.erb:15]: Hey @ievgend2 there are some users that scrape this page for the links, could you still include this html? You can hide it using css like `display: none`

**Review by micronix — CHANGES_REQUESTED** [review]
I think it all looks good, the only thing is that we don't want to completly remove the html for the downloads on each page snapshots, pipefiles, covid but we can hide them right now.

**Comment by ievgend2** [comment]
New Layout:
image

**Pull request merged**